analyzer/codechecker_analyzer/analyzers/analyzer_clang_tidy.py

```diff
@@ -117,7 +117,8 @@ def construct_analyzer_cmd(self, result_handler):
             analyzer_cmd.append("-checks='%s'" % checkers_cmdline.lstrip(','))
 
             LOG.debug(config.analyzer_extra_arguments)
-            analyzer_cmd.append(config.analyzer_extra_arguments)
+            if config.analyzer_extra_arguments:
```

Comment:
Is `config.analyzer_extra_arguments` a list of arguments or only a string of arguments, maybe we should change this to be a list too?